pkg/services/sqlstore/permissions/dashboard.go

```diff
@@ -111,14 +111,21 @@ func (f *accessControlDashboardPermissionFilter) Where() (string, []any) {
 }
 
 func (f *accessControlDashboardPermissionFilter) buildClauses() {
-	if f.user == nil || f.user.Permissions == nil || f.user.Permissions[f.user.OrgID] == nil {
+	if f.user == nil || f.user.IsNil() || f.user.GetPermissions() == nil || len(f.user.GetPermissions()) == 0 {
```

Comment:
GetPermissions should never be nil but it doesn't hurt to check

Reply:
I'm removing this in order to be consistent